### PR TITLE
RNDV-1620 added the videos to the guide

### PR DESCRIPTION
**Pull request by DaliNeji**

### CODE REVIEW

File: docs/user-guide/index.fr.md [hunk 1]
```diff
@@ -78,6 +109,13 @@ Vous pouvez utiliser des filtres tels que la localisation, le prix, le rayon, la
...
 </p>
 
+<div style="display: flex; justify-content: center;">
+<video controls width=" 310" height="480 ">
+<source src="./Videos/FR/Final-product/AdvancedSearch.mp4" type="video/mp4">
+</video>
+</div>
+
+
 ## Vérification des attributs de la boutique
```
Comment (safa-rekik) [inline docs/user-guide/index.fr.md:119]: Please change this title "Consulter les détails de la boutique"

File: docs/user-guide/index.fr.md [hunk 2]
```diff
@@ -36,6 +43,17 @@ Voici trois méthodes :
...
     - Cliquez sur "Continuer avec email" (vous devez avoir déjà créé un compte)
+  
+  <div style="display: flex; justify-content: center;">
+<video controls width=" 310" height="480 ">
+<source src="./Videos/FR/Final-product/Login_1.mp4" type="video/mp4">
+</video>
+</div>
+<div style="display: flex; justify-content: center;">
+<video controls width=" 310" height="480 ">
+<source src="./Videos/FR/Final-product/GoogleLogin.mp4" type="video/mp4">
```
Comment (safa-rekik) [inline docs/user-guide/index.fr.md:54]: make sure to put the google login video under the google login title, and login with email cideo aunder login with email video

**Review by safa-rekik — CHANGES_REQUESTED** [review]
Concerning the full experience video, Mohammad asked me to put it in the  "quick start" page.

**Review by safa-rekik — APPROVED** [review]

**Comment by safa-rekik** [comment]
@yregaieg Hi, I couldn't merge this ticket, I think because you requested a change, please approve and merge

**Review by yregaieg — APPROVED** [review]
Still not really happy with the way this is implemented, but I am approving to not stall the merge

**Pull request merged**